test/initiator_group_test.go

```diff
@@ -94,9 +113,49 @@ func (suite *IGWorkflowSuite) TestCreateIGIscsiInitiators() {
 	}
 	_, err := suite.initiatorGrpService.CreateInitiatorGroup(newIG)
 	assert.Nilf(suite.T(), err, "Unable to create initiator group, err: %v", err)
+	// Update initiator group
+	var ipAdd *string = param.NewString("10.1.0.0")
```

Comment:
This could be simplified to:

   ipAdd := "10.1.0.0"

And then below, change ipAdd to &ipAdd and change *ipAdd to ipAdd.

Reply:
That should fine at least here since param.NewString is being leveraged to get the pointer to a string in one line. I am glad to see param package is being used which I will encourage while writing SDK-related code. thought ?

Reply:
Line 120 and others show appropriate usage of param.NewString.  No need to overuse it.